stats/src/main/java/io/airlift/stats/JmxGcMonitor.java

```diff
@@ -140,6 +147,20 @@ private synchronized void onNotification(Notification notification)
                 long applicationRuntime = max(0, info.getStartTime() - lastGcEndTime);
                 lastGcEndTime = info.getEndTime();
 
+                if (info.getBeforeGcTotal().toBytes() / heapMemoryLimit > .8) {
```

Comment:
Nit: we prefer to write decimals with leading zero `0.8`

Reply:
done